napari/layers/base/base.py

```diff
@@ -1257,7 +1307,14 @@ def displayed_coordinates(self):
         coordinates = self.world_to_data(self._position)
         return [coordinates[i] for i in self._dims_displayed]
 
-    def get_status(self, position, *, world=False):
+    def get_status(
+        self,
+        position: np.ndarray,
+        *,
+        view_direction: np.ndarray,
```

Comment:
Should this be optionally None, for 2D?

Reply:
I also think array-likes (e.g. list of float) should be allowed.

btw, please update the docstring. ;)